azure-pipelines-ci.yml

```diff
@@ -31,6 +31,7 @@ extends:
       - stage: main
         jobs:
           - job: BuildTestLint
+            timeoutInMinutes: 120
```

Comment:
can we lower it to 90? that should be enough ? 👌

Reply:
I hope so 😆